.gitlab-ci.yml

```diff
@@ -0,0 +1,21 @@
+# Barebones template to ensure pipeline is triggered
+include:
+  - project: 'analytics/artificial-intelligence/ai-platform/aip-infrastructure/ci-templates/ci-cd-template'
+    ref: '024dcb2d169eeea2b3149f8bdeb3cf58f61e7ade'  # Pin to a commit/branch to prevent unwanted changes occurring.
+    file: 'environments/devex.yml'
+  - project: 'analytics/artificial-intelligence/ai-platform/aip-infrastructure/ci-templates/ci-cd-template'
+    ref: '024dcb2d169eeea2b3149f8bdeb3cf58f61e7ade'  # Pin to a commit/branch to prevent unwanted changes occurring.
+    file: 'blocks/aws.yml'
+
+build:
+  extends: 
+    - .devex_internal_eks
+    - .generate_kubeconfig
+  variables:
+    PIPELINE_VERSION: "1.0"
+  stage: build
+  script:
+    - mv /root/.kube .
```

Comment:
TODO: try to mount this to container

Reply:
If you look AlexL's CICD code, does it mount the `.kube` directory?